pkg/cephmgr/mon/config.go

```diff
@@ -148,6 +148,15 @@ func GenerateConfigFile(context *clusterd.Context, cluster *ClusterInfo, pathRoo
 		return "", fmt.Errorf("failed to add initial monitor config sections, %+v", err)
 	}
 
+	// if there's a config file override path given, process the given config file
+	if context.ConfigFileOverride != "" {
+		err := configFile.Append(context.ConfigFileOverride)
```

Comment:
the Append will handle all merges of the config files with different sections, and overriding settings in each section? sounds perfect